### PR TITLE
Remove extracted cops in `Capybara`, `FactoryBot` and `Rails` departments.

### PR DESCRIPTION
**Pull request by ydah**
This PR remove extracted cop departments. (`Capybara`, `FactoryBot`, `Rails`)
______________________________________________________________________

Before submitting the PR make sure the following are checked:

- [x] Feature branch is up-to-date with `master` (if not - rebase it).
- [x] Squashed related commits together.
- [-] Added tests.
- [x] Updated documentation.
- [x] Added an entry to the `CHANGELOG.md` if the new code introduces user-observable changes.
- [x] The build (`bundle exec rake`) passes (be sure to run this locally, since it may produce updated documentation that you will need to commit).

### CODE REVIEW

File: config/default.yml [hunk 1]
```diff
@@ -999,210 +999,10 @@ RSpec/Capybara:
...
   Language: *2
 
-RSpec/Capybara/CurrentPathExpectation:
-  Description: Checks that no expectations are set on Capybara's `current_path`.
-  Enabled: true
-  VersionAdded: '1.18'
-  VersionChanged: '2.0'
-  Reference: https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/Capybara/CurrentPathExpectation
-
 RSpec/Capybara/FeatureMethods:
```
Comment (pirj) [inline config/default.yml:1002]: Can we replace featuremethods with RSpec/Dialect configuration?
Would close a bunch of issues, too:

Comment (bquorning) [reply config/default.yml:1002]: Good idea @pirj. I’ve added this as [an issue in the 3.0 milestone](https://github.com/rubocop/rubocop-rspec/issues/1852).

**Comment by bquorning** [comment]
@ydah Now that we have a separate `2-x-stable` branch, I think we are ready to merge this pull request after the conflicts are fixed.

File: docs/modules/ROOT/pages/departments_version_2.adoc [hunk 2]
```diff
@@ -1,6 +1,6 @@
-= Departments
+= Departments in Version 2.x
 
-RuboCop RSpec currently has four departments:
+RuboCop RSpec Version 2.x has four departments:
```
Comment (bquorning) [inline docs/modules/ROOT/pages/departments_version_2.adoc:3]: We have the upgrade document in docs/modules/ROOT/pages/upgrade_to_version_3.adoc, so I don’t think we need to document which departments were available in version 2. Let’s just document the departments in ~v3~ master branch.

Comment (ydah) [reply docs/modules/ROOT/pages/departments_version_2.adoc]: I updated this PR. Thank you!

**Comment by ydah** [comment]
I updated this PR.

**Review by bquorning — APPROVED** [review]
👏🏼

**Comment by ydah** [comment]
It is also a blocker for subsequent PR, so this PR will be merged. We can also follow up.

**Pull request merged**